### PR TITLE
Feature a11y

### PR DESCRIPTION
**Pull request by Meggie-H**

### CODE REVIEW

File: src/components/CommitGraph.tsx [hunk 1]
```diff
@@ -35,21 +35,39 @@ const CommitGraph = () => {
...
+        labels: {
+          color: '#edf2f7',
+        },
+      },
+    },
+  };
+
   if (ContributorQuery.isLoading) {
-    return <div>Loading...</div>;
+    return (
```
Comment (Hugovs2000) [inline src/components/CommitGraph.tsx:55]: Noticed you are doing a few skeletons. You can maybe extract them out and make a shared skeleton component

File: src/components/GitTable.tsx [hunk 2]
```diff
@@ -38,7 +42,7 @@ function GitTable() {
               <div className="align-center flex">
                 <img
                   src={commit.author?.avatar_url}
-                  alt="Avatar"
+                  alt={`${commit.author?.login} profile picture`}
```
Comment (Hugovs2000) [inline src/components/GitTable.tsx]: Maybe do this for safety if for some reason there is no author 
```
alt={`${commit.author?.login ?? "User"} profile picture`}
```

**Review by Hugovs2000 — CHANGES_REQUESTED** [review]
Looking good Megs, just a few comments on the skeletons and a few safety check suggestions.

File: src/components/CommitGraph.tsx (hunk 1 above)
Comment (Meggie-H) [reply src/components/CommitGraph.tsx:55]: How would that work?

Comment (Meggie-H) [reply src/components/CommitGraph.tsx:55]: Please have a look at it again

File: src/components/CommitGraph.tsx [hunk 3]
```diff
@@ -35,21 +35,39 @@ const CommitGraph = () => {
...
 
   if (ContributorQuery.isError) {
     return <div>Error fetching commit data</div>;
   }
 
   return (
-    <div className="rounded-lg bg-gray-900 p-4">
-      <PolarArea data={chartData} />
+    <div className="flex h-full flex-col items-center rounded-2xl bg-gray-900 p-4">
+      <h2 className="width-full text-gray-200">Commits</h2>
```
Comment (yanevdw) [inline src/components/CommitGraph.tsx]: Should this not be `w-full`?

File: src/components/LanguageGraph.tsx [hunk 4]
```diff
@@ -35,8 +47,8 @@ const LanguageGraph = () => {
 
   return (
     <div className="flex flex-col items-center rounded-2xl bg-gray-900 p-4">
-      <h2 className="width-full">Languages</h2>
-      <Doughnut data={data} />
+      <h2 className="width-full text-gray-200">Languages</h2>
```
Comment (yanevdw) [inline src/components/LanguageGraph.tsx]: `w-full` here again

File: src/components/NotFoundPage.tsx [hunk 5]
```diff
@@ -0,0 +1,19 @@
+import { Link } from '@tanstack/react-router';
+import React from 'react';
+
+const NotFoundPage = () => {
+  return (
+    <div className="bg-950 flex h-screen w-screen flex-col items-center justify-center text-gray-200">
```
Comment (yanevdw) [inline src/components/NotFoundPage.tsx:6]: Is the `bg-950` a valid tailwind class? If it is, then just disregard this.

**Review by yanevdw — APPROVED** [review]
Excellent work! Just check the comments about some tailwind classes.

**Review by yanevdw — APPROVED** [review]
Styling has been fixed. Looks good to me.

File: src/components/CommitGraph.tsx (hunk 1 above)
Comment (Hugovs2000) [reply src/components/CommitGraph.tsx:55]: So what I meant was you can have a component called `<SharedSkeleton />` for example where your loading code can go. I.E 
```
<div className="flex h-full w-full flex-col items-center rounded-2xl bg-gray-900 p-4">
        <div className="skeleton h-full w-full bg-gray-800"></div>
</div>
``` 
can go in there. Then you have a reusable loading component

**Review by Musa-Mabasa — APPROVED** [review]
I didn't see any issues that were not already commented on. Good job 👍

**Review by LesediMoko — APPROVED** [review]
Looks good, Meg!

**Pull request merged**